### DISCUSSION TITLE
Welcome to enphase-dashboard Discussions!

### DISCUSSION THREAD

**Discussion by SCHibbard** [SCHibbard/enphase-dashboard#7] [Announcements]
Please provide feedback, suggestions and feedback here!

**Comment by SCHibbard** [comment]
@khorton: thanks for all the feedback. Realized I had never turned discussions on.  Question: what OS are you using on your system? Interested in knowing if it runs ok on boxes with something other than Ubuntu.

**Reply by khorton** [reply]
Running Ubuntu 22.04 LTS, as the purpose of this server is to run various things under Docker, and I wanted to be on a well trodden path to avoid as many bizarre issues as possible
